### PR TITLE
minor updates

### PR DESCRIPTION
**Pull request by sptl-tech**
- Removal of old pledge info
- adjustment to pledge date

### CODE REVIEW

File: frontend-v2/src/components/CompanyCarousel.js [hunk 1]
```diff
@@ -33,6 +33,11 @@ const CompanyCarousel = () => {
         className={carouselStyles.company__logo}
         alt="company-logo"
       />
+      <img
+        src={nfl}
+        className={carouselStyles.company__logo}
+        alt="company-logo"
+      />
```
[the remark this answers is not in the document]
Comment (sptl-tech) [reply frontend-v2/src/components/CompanyCarousel.js]: Resolved

**Comment by sptl-tech** [comment]
Things changed: 

- Changed placement of NFL logo to stay in alpha order 
- Minor cosmetic changes and fixed typos 
- Added a section in the about page for crediting photographers 
- Adjusted 2 early alum members in Alumni.json to reflect them as graduated and not inactive

**Comment by sptl-tech** [comment]
Things changed:

Changed placement of NFL logo to stay in alpha order
Minor cosmetic changes and fixed typos
Added a section in the about page for crediting photographers
Adjusted 2 early alum members in Alumni.json to reflect them as graduated and not inactive

**Comment by williamtnguyen** [comment]
@sptl-tech Already addressed in #109, so gonna close this. If you want to add photographer credit then please fork a new branch from the current master branch on this repo & rebase the photo-credit changes onto it. Or maybe get new webmaster to do it haha

**Pull request closed without merging**